api-client/src/endpoints/__tests__/createFeedbackEndpoint.spec.ts

```diff
@@ -1,7 +1,3 @@
-/**
- * only needed as FormData is a web-specific type
- * @jest-environment jsdom
- */
```

Comment:
jest-environment jsdom has been removed from jest. So to avoid adding another dependency I mocked FormData instead.